sn_client/src/wallet.rs

```diff
@@ -35,6 +35,7 @@ pub struct WalletClient {
 
 impl WalletClient {
     /// Create a new wallet client.
```

Comment:
Can this text here also go in the markdown document? Sorry, I've never used this feature before.

Reply:
Yes, if this is agreed upon it will be the same for all documentation.
I left it in the rs file because it gives a quick overview to the reader (leaving the md for more detail)